Dockerfile

```diff
@@ -1,114 +1,122 @@
-# IMPORTANT NOTE: To avoid compliation and runtime errors, this ALPINE_VERSION needs to match the alpine version of the stage `production_build`.
-# As the the stage `production_build` uses the docker image `elixir:1.13.1-alpine` as base image it is not clear which alpine version is used here.
-# Unfortunately, the maintainers of docker image do not provide the possibility to pin the alpine version of the image.
-# However, they also change the underlying alpine version from time to time. This can cause compilation and runtime errors,
-# e.g. https://stackoverflow.com/questions/72609505/issue-with-building-elixir-and-beam-on-alpine-linux .
-# 
-# This means, if a compliation and runtime error occur, we need to look up the alpine version directly in the codebase and match it by hand: 
-# - https://github.com/erlef/docker-elixir/blob/master/1.11/alpine/Dockerfile
-# - https://github.com/erlang/docker-erlang-otp/blob/master/23/alpine/Dockerfile
-ARG ALPINE_VERSION=3.16
-
-FROM elixir:1.13.1-alpine as elixir_alpine
-
-ENV APP_PATH=/app
-
-RUN set -eux ; \
-    apk add \
-      --update-cache \
-      --no-cache \
-      --repository http://dl-cdn.alpinelinux.org/alpine/v3.11/main/ \
-      # Version taken from https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.11
-      nodejs=12.22.6-r0 \
-      # Version taken from https://pkgs.alpinelinux.org/packages?name=npm&branch=v3.11
-      npm=12.22.6-r0
-
-RUN mix do local.hex --force, local.rebar --force
-
-WORKDIR $APP_PATH
-
-FROM elixir_alpine as development
-
-RUN set -eux ; \
-    apk add \
-      # The package `inotify-tools` is needed for instant live-reload of the the phoenix server
-      inotify-tools \
-      postgresql-client
+# Find eligible builder and runner images on Docker Hub. We use Ubuntu/Debian instead of
+# Alpine to avoid DNS resolution issues in production.
+#
+# https://hub.docker.com/r/hexpm/elixir/tags?page=1&name=ubuntu
+# https://hub.docker.com/_/ubuntu?tab=tags
+#
+#
+# This file is based on these images:
```

Comment:
Can you please explain in more detail why we are basing the docker images on `hexpm/elixir` and not the official `elixir` docker images.

I have ideas why we do this, but it is better to document them for future reference.